From 422b375035ccc5aba6c4f5207d3aa5ba445265eb Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:05:26 -0400
Subject: [powerpc] scom: Use "devspec" rather than "path" in debugfs entries

Message-id: <1410545655-205645-98-git-send-email-dzickus@redhat.com>
Patchwork-id: 94245
O-Subject: [RHEL7 PATCH 097/626] powerpc/scom: Use "devspec" rather than "path" in debugfs entries
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 81fafea67a2ea418673df2df2b22a81f0c5a455a
Author: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Date:   Thu Aug 29 17:25:35 2013 +1000

    powerpc/scom: Use "devspec" rather than "path" in debugfs entries

    This is the traditional name for device-tree path, used in sysfs,
    do the same for the XSCOM debugfs files.

    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/sysdev/scom.c b/arch/powerpc/sysdev/scom.c
index cb20d54..3963d99 100644
--- a/arch/powerpc/sysdev/scom.c
+++ b/arch/powerpc/sysdev/scom.c
@@ -180,7 +180,7 @@ static int scom_debug_init_one(struct dentry *root, struct device_node *dn,
 
  debugfs_create_file("addr", 0600, dir, ent, &scom_addr_fops);
  debugfs_create_file("value", 0600, dir, ent, &scom_val_fops);
- debugfs_create_blob("path", 0400, dir, &ent->blob);
+ debugfs_create_blob("devspec", 0400, dir, &ent->blob);
 
  return 0;
 }
-- 
1.7.1